src/openklant/components/contactmomenten/swagger2.0.json

```diff
@@ -168,6 +168,34 @@
                             "-medewerker_identificatie"
                         ]
                     },
+                    {
+                        "name": "medewerkerIdentificatie__identificatie",
+                        "in": "query",
```

Comment:
Joeri en Sergei hadden dit onderling besproken, en zouden het chique vinden als we afwijkingen met ***afwijking** in de OAS opnemen. Ondanks je handige overzicht op de wiki, Sergei was met name bang dat die door niemand zou worden gevonden. Speelt voor openklant een stuk minder dan je open-zaak aanpassingen

Reply:
ik ga dit in een aparte PR doen, en dan neem ik alle API spec changes mee